sections/Carousel/CardCarousel.tsx

```diff
@@ -0,0 +1,134 @@
+import { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import Slider from "../../components/ui/Slider.tsx";
+import { ButtonColor, colorClasses } from "../../constants.tsx";
+import { useId } from "../../sdk/useId.ts";
+import { clx } from "../../sdk/clx.ts";
+
+export interface CardColors {
+  /** @title Cor dos textos */
+  textColor?: string;
+  /** @title Cor de fundo do card*/
+  bgcolor?: string;
+  /** @title Cor de fundo do botão*/
+  buttonColor?: ButtonColor;
+}
+
+/** @titleBy title */
+export interface Card {
+  /** @title Texto da flag acima do título */
+  flag?: string;
+  /** @title Título */
+  title: string;
+  /** @title Descrição */
+  /** @format textarea */
+  description?: string;
```

Comment:
Isso está funcionando no admin? geralmente fazem tudo num jsdoc apenas
```suggestion
  /**
   * @title Descrição
   * @format textarea
   */
  description?: string;
```

Reply:
estava funcionando sim, mas ajustei de qualquer forma.